ads/mediaimpact.js

```diff
@@ -39,7 +48,7 @@ export function mediaimpact(global, data) {
     view: 'm',
     async: true,
   };
-  loadScript(global, 'https://ec-ns.sascdn.com/diff/251/divscripte/amp.js?dom=' + window.context.location.host, () => {
+  loadScript(global, 'https://ec-ns.sascdn.com/diff/251/pages/amp_default.js', () => {
```

Comment:
Not sure I fully understand you. Prefetch does help you speed up a little bit. Of course it's totally optional, if you want to reduce the network pings.